### DISCUSSION TITLE
Loading pre-trained models for finetuning

### DISCUSSION THREAD

**Discussion by ipiriyan2002** [open-mmlab/mmocr#1965] [General]
I was wondering how I can load pre-trained weights to fine-tune the model to my custom dataset. Any help would be appreciated.

**Comment by gaotongxiao** [comment]
You can specify `load_from` in the configuration file and point the configuration file to your dataset. See https://mmocr.readthedocs.io/en/dev-1.x/user_guides/config.html#checkpoint-loading-configuration
